code/modules/mob/living/carbon/human/species_types/angel.dm

```diff
@@ -141,4 +141,4 @@
 		H.movement_type &= ~FLYING
 		override_float = FALSE
 		H.pass_flags &= ~PASSTABLE
-		H.CloseWings()
\ No newline at end of file
+		H.CloseWings()

```

Comment:
Delete this change

Reply:
Dont know how to fix my computer is broken so been using the ingithub stuff and everytime I remove the space it readds it

Reply:
but yeah if your able to can you fix it for me? if not just close this pr and I will open a new one that does not mess with the angel and synth files.